.github/workflows/go.yml

```
@@ -336,19 +337,21 @@ jobs:
       - name: Setup cache key
         run: cp ${{matrix.package}}/go.mod go.mod -v
 
+      - name: Read golangci-lint version
+        id: golangci_version
+        run: echo "version=v$(cat .golangci-version)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:45: Double quote to prevent globbing and word splitting

(shellcheck)